charts/cp-control-center/values.yaml

```diff
@@ -61,6 +61,10 @@ nodeSelector: {}
 ## Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
 tolerations: []
 
+## Pod scheduling constraints
+## Ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity
+affinity: {}
```

Comment:
Note that the `cp-control-center`'s deployment template already includes logic to conditionally use the `Values.affinity` config, however the `affinity` block is missing from this `values.yaml` file.